### DISCUSSION TITLE
Disable Copilot button on status bar doesn't work

### DISCUSSION THREAD

**Discussion by jackHedaya** [community/community#33290] [Copilot Conversations] [answered]
Its very frustrating as this makes me have to disable copilot for every workspace related to school. Can someone please advise?

**Comment by Akash1134** [comment, marked as answer]
Hi @jackHedaya , please check that your computer is not set to Do Not Disturb mode while using Copilot. Thanks!

**Comment by jackHedaya** [comment]
Solved! VSCode was in DND.

On Mon, Sep 19, 2022 at 8:33 AM Akash ***@***.***> wrote:

> Hi @jackHedaya <https://github.com/jackHedaya> , please check that your
> computer is not set to Do Not Disturb mode while using Copilot. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/33290#discussioncomment-3680268>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEZ47UUZMZV2BZDJAKC5LITV7BMQZANCNFSM6AAAAAAQOSZTWE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Best
Jack

**Comment by jackHedaya** [comment]
Thank you!

On Mon, Sep 19, 2022 at 11:19 AM Jack Hedaya ***@***.***> wrote:

> Solved! VSCode was in DND.
>
> On Mon, Sep 19, 2022 at 8:33 AM Akash ***@***.***> wrote:
>
>> Hi @jackHedaya <https://github.com/jackHedaya> , please check that your
>> computer is not set to Do Not Disturb mode while using Copilot. Thanks!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/orgs/community/discussions/33290#discussioncomment-3680268>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AEZ47UUZMZV2BZDJAKC5LITV7BMQZANCNFSM6AAAAAAQOSZTWE>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> Best
> Jack
>


-- 
Best
Jack
